docs/development/serverless.md

```diff
@@ -0,0 +1,74 @@
+# Serverless
+
+Serverless is a powerful and popular model where you don’t have to worry about managing and maintaining your application infrastructure. There are two main paradigms when it comes to Serverless.  The first is the operations paradigm which is responsible for the orchestration of containers to zero.
+
+The second paradigm is functions.  In the serverless context, a function is a single-purpose piece of code created by the developer but run and monitored by the managed infrastructure. A serverless function’s value is its simplicity and swiftness, which can entice even those who don’t consider themselves developers.
```

Comment:
```suggestion
The second paradigm is the functions programming model.  In the serverless context, a function is a single-purpose piece of code created by the developer but run and monitored by the managed infrastructure. A serverless function’s value is its simplicity and swiftness, which can entice even those who don’t consider themselves developers.
```
There are a lot of other benefits as well (at least with Knative Functions), such as language packs, which allow enterprises to create custom function templates for their developers, expanding runtime options and locking in approved dependencies. Probably too much to get into in this short intro, but something to think about for a future revision.